src/components/FeaturedPosts.js

```diff
@@ -0,0 +1,157 @@
+import Slider from 'react-slick';
+import {css} from '@emotion/core';
+import {map} from 'ramda';
+import {mapNodeToProps} from '~/utilities';
+import {LOCHMARA_COLOR, mq} from '~/constants';
+import Img from 'gatsby-image';
+import {ExternalLink, Text} from '~/components';
+import {Link} from 'gatsby';
+
+const settings = {
+  adaptiveHeight: true,
+  centerMode: false,
+  dots: true,
+  infinite: true,
+  speed: 750,
+  slidesToShow: 1,
+  slidesToScroll: 1,
+  autoplay: true,
+  autoplaySpeed: 10000,
+  cssEase: 'ease',
+};
+
+// TODO: dynamically determine height
+const containerStyles = css`
```

Comment:
Similar question to the one above, can we move this to a `styles` file for this component? It'll clean up the code a little bit if we had this as a separate file I feel. WDYT?

Reply:
Completed this todo (kind of)... will get rid

Reply:
We could move it to a styles file... not sure if we should though. It's not overwhelming as it is. Your call.